### PR TITLE
Update README to remove "we no longer maintain a support server" things

### PR DESCRIPTION
**Pull request by zNeolythic**
uh yeah title says it all

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,17 +13,17 @@ You can invite the official bot [here](https://discord.com/api/oauth2/authorize?
...
+For help installing the bot, questions, suggestions to improve or contributions, feel free to join the official community of developers and self-hosters!
 
 [![Discord server](https://discord.com/api/guilds/1255250024741212262/embed.png?style=banner3)](https://discord.gg/PKKhee4fvy)
 
 > You can also find the server of our official Ballsdex bot here:
 > 
 > [![Discord server](https://discord.com/api/guilds/1049118743101452329/embed.png?style=banner2)](https://discord.gg/Qn2Rkdkxwc)
 
-## Suggestions, questions and bug reports
+## Issues and bug reports
```
Comment (flaree) [inline README.md:24]: Suggestions are still welcome on the repo

**Review by flaree — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (flaree) [reply README.md:24]: Other than this, everything else is fine to merge

**Comment by zNeolythic** [comment]
alright, added

**Comment by flaree** [comment]
Thanks for making that change, everything's good now 😁

**Pull request merged**